### ISSUE TITLE
Navbar text in combination with navbar-right has the wrong margin-right.

### ISSUE THREAD

**Issue by brushmate** [twbs/bootstrap#16046]
From v3.3.0 on texts in navbars that have `navbar-right` applied have a wrong margin on the right side. I have created two fiddles. The [first fiddle][1] uses v3.3.0 and you can see that there is no margin on the right side of the text. The [second fiddle][2] uses v3.2.0 and the margin is correct.

Was this behaviour changed on purpose? I could not find anything regarding this issue in the changelog.

  [1]: http://jsfiddle.net/mc7jfktv/
  [2]: http://jsfiddle.net/gbs4d2wr/

**Comment by twbs-lmvtfy** [comment]
Hi @brushmate!

You appear to have posted a live example (http://fiddle.jshell.net/mc7jfktv/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 15, column 7: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.2 ; saw what appears to be usage of Bootstrap 3.3.0

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by twbs-lmvtfy** [comment]
Hi @brushmate!

You appear to have posted a live example (http://fiddle.jshell.net/gbs4d2wr/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 15, column 7: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.2 ; saw what appears to be usage of Bootstrap 3.2.0

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by kkirsche** [comment]
@brushmate this seems to me to be a duplicate of #15706 which corresponds with pull request #15744

**Comment by brushmate** [comment]
@kkirsche You are right. Sorry, I did not see this issue.

**Comment by kkirsche** [comment]
No worries! Thanks for reporting it though :smiley:
